src/ipsis/course-merge/CourseMergeOfferingCollectionEntity.js

```diff
@@ -3,14 +3,13 @@
  * See: ISirenCourseMergeSerializer.SerializeCourseOfferingListResult
  */
 import { Entity } from '../../es6/Entity.js';
-import { Rels } from '../../hypermedia-constants.js';
 
 export class CourseMergeOfferingCollectionEntity extends Entity {
 	courseMergeOfferings() {
 		if (!this._entity) {
 			return;
 		}
-		return this._entity.getSubEntitiesByRel(Rels.IPSIS.SISCourseMerge.courseMergeOffering);
+		return this._entity.entities;
```

Comment:
This change is needed due to how the `siren-parser` `Entity` (`_entity`) works. As part of it's constructor, it parses `entities` into objects like `entitiesByRel` for easier lookup later. Because of this, updating `entities` doesn't update `entitiesByRel` which means `getSubEntitiesByRel` will still only return for the current page instead of `entities`. And the only time we call the `Entity` constructor is when we receive the new `href`.

Reply:
I see why we wouldn't like the change, but we should never be returning anything else in the subentities so I feel like this would've been ok from the start.